extensions/data-workspace/src/dialogs/newProjectDialog.ts

```diff
@@ -51,13 +48,17 @@ export class NewProjectDialog extends DialogBase {
 				this.showErrorMessage(constants.ProjectDirectoryAlreadyExistError(this.model.name, this.model.location));
 				return false;
 			}
-
-			return true;
 		}
 		catch (err) {
 			this.showErrorMessage(err?.message ? err.message : err);
 			return false;
 		}
+
+		if (await this.workspaceService.validateWorkspace() === false) {
```

Comment:
any reason for this not to be in the try too?

Reply:
ot really - wasnt before so didnt put it.